### DISCUSSION TITLE
DeepMD kit GPU version performance

### DISCUSSION THREAD

**Discussion by PolyuWeldingSpock** [deepmodeling/deepmd-kit#2657] [Q&A]
Hi everyone, I am new to DeepMD kit and I have some questions about installing and optimizing its performance.

1. I have already installed CUDA v11.8 on my computer. Then I followed the instructions provided in this link: https://github.com/deepmodeling/deepmd-kit/blob/master/doc/install/easy-install.md and ran the command conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.6 horovod -c https://conda.deepmodeling.com -c defaults to install DeepMD kit. Will there be any configuration issues between different CUDA versions? Or if I have already installed CUDA 11.8, can I simply run the command conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps horovod -c https://conda.deepmodeling.com -c defaults to install DeepMD kit without additional cuda installzation but with GPU support that matches my local CUDA version? I would greatly appreciate any help in resolving this issue.

**Comment by njzjz** [comment]
Local CUDA libraries cannot be used for conda packages.
